resources/resinos.robot

```diff
@@ -40,6 +40,23 @@ File list "@{files_list}" does not exist in "${mount_destination}"
     : FOR   ${files_line}     IN  @{files_list}
     \   File Should Not Exist  ${mount_destination}${files_line}  msg="Backup file ${files_line} found on the rootfs that should not exist." 
     
+Clone "${git_url}" in "${dir}"
+    ${result} =  Run Process    git clone ${git_url}    shell=yes   cwd=${dir}
+    Process ${result}
+
+Set up getty service for "${device_type}"
+    ${result} =  Run Process    ./serial-it.sh --root-mountpoint ${mount_destination} -b ${device_type}     shell=yes   cwd=${tmp_dir}/serial-it
+    Process ${result}
+
+Run minicom "${script}" at startup and capture "${output}"
```

Comment:
I think we can remove at startup part of this - Run minicom "${script}" and output to "${output}"